packages/widgetbook/lib/src/knobs/slider_knob.dart

```diff
@@ -0,0 +1,152 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'package:widgetbook/src/knobs/knobs.dart';
+import 'package:widgetbook/src/knobs/nullable_checkbox.dart';
+
+class SliderKnob extends Knob<double> {
+  SliderKnob({
+    required String label,
+    String? description,
+    required double value,
+    this.max = 1,
+    this.min = 0,
+    this.divisions,
+  }) : super(
+          label: label,
+          description: description,
+          value: value,
+        );
+
+  final double max;
+  final double min;
+  final int? divisions;
+
+  @override
+  Widget build() => SliderKnobWidget(
+        label: label,
+        description: description,
+        value: value,
+        min: min,
+        max: max,
+        divisions: divisions,
+      );
+}
+
+class NullableSliderKnob extends Knob<double?> {
+  NullableSliderKnob({
+    required String label,
+    String? description,
+    required double value,
+    this.max = 1,
+    this.min = 0,
+    this.divisions,
+  }) : super(
+          label: label,
+          description: description,
+          value: value,
+        );
+
+  final double max;
+  final double min;
+  final int? divisions;
+
+  @override
+  Widget build() => SliderKnobWidget(
+        label: label,
+        description: description,
+        value: value,
+        min: min,
+        max: max,
+        divisions: divisions,
+        nullable: true,
+      );
+}
+
+class SliderKnobWidget extends StatefulWidget {
+  const SliderKnobWidget({
+    Key? key,
+    required this.label,
+    required this.description,
+    required this.value,
+    required this.max,
+    required this.min,
+    required this.divisions,
+    this.nullable = false,
+  }) : super(key: key);
+
+  final String label;
+  final String? description;
+  final double? value;
+
+  final double max;
+  final double min;
+  final int? divisions;
+  final bool nullable;
+
+  @override
+  State<SliderKnobWidget> createState() => _SliderKnobWidgetState();
+}
+
+class _SliderKnobWidgetState extends State<SliderKnobWidget> {
+  double _value = 0;
+  @override
+  void initState() {
+    super.initState();
+    if (widget.value != null) {
+      _value = widget.value!;
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final disabled = widget.value == null;
+    return KnobWrapper(
+      description: widget.description,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceAround,
+        children: [
+          Center(
+            child: Text(
+              '${widget.label}:',
+              overflow: TextOverflow.ellipsis,
+            ),
+          ),
+          const SizedBox(
+            width: 10,
+          ),
+          SizedBox(
+          width: 45,
+            child: Text(
+              _value.toStringAsFixed(2),
+            ),
+          ),
```

Comment:
Due to the hard coded width this overflows if the `value` becomes larger (in this case >100): 

<img width="390" alt="image" src="https://user-images.githubusercontent.com/16862002/163112697-3f611c4c-7e5f-4ad8-8b03-2959dabe981f.png">

Reply:
I think this should be fixed when I implement it to follow the new design you gave!